packages/foreman/rubygem-sassc/rubygem-sassc.spec

```diff
@@ -15,9 +14,11 @@ Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
 Requires: ruby >= 2.0.0
 BuildRequires: ruby-devel >= 2.0.0
 BuildRequires: rubygems-devel
-BuildRequires: rubygem(ffi) >= 1.9
-BuildRequires: rubygem(ffi) < 2
+BuildRequires: (rubygem(ffi) >= 1.9 with rubygem(ffi) < 2)
 # end specfile generated dependencies
+# the upstream source builds an embedded libsass
+# but we use the system one, which makes this package noarch
+BuildArch: noarch
 
 BuildRequires: (libsass.so.1()(64bit) if libc.so.6()(64bit))
 BuildRequires: (libsass.so.1 if libc.so.6)
```

Comment:
The idea was that this would be for non-x86_64 architectures as well, but I may be mistaken in that because at least for AARCH64 on Fedora 40 it also provides the 64 bit version.
```console
$ rpm -q --provides glibc-2.38.9000-29.fc40.aarch64.rpm | grep libc.so.6
libc.so.6()(64bit)
```


```suggestion
```

And then also remove the `Requires`.

Reply:
Or just simplify it to `libsass.so.1` since that's always provided. EL 8+ are never built for i686 only so I don't think we'll ever need to worry about i686.

Reply:
One thing came to my mind: today this package is arch dependent (as the generation thinks it has an extension, even tho we patch it out).
If we keep it like this, we can drop the 32 bit things as they will never match arch wise (like you suggest), but we *also* can go ahead and use %isa to print the real arch for the dep on libsass.

Reply:
I'd prefer to make the package `noarch` (since it is). But then you can't use `%isa` because that's not defined on `noarch` packages.